README.md

```diff
@@ -145,21 +148,26 @@ does not have a `[[Call]]` internal method; it is not possible to invoke the
 When called with arguments `value1` and `value2`:
 
 * If `Type(value1)` is not `Object` return `false`.
-* Let `B` be `? value1.[[GetOwnProperty]](@@builtin)`.
-* If `B` is `undefined`, return `false`.
-* Let `V1` be `? GET(value1, @@builtin)`.
-* If `Type(V1)` is not `String`, return `false`.
+* If `? [[HasOwnProperty]](value1, @@builtin)` is `false`, return `false`.
```

Comment:
I think this might be much more clear if it was in a separate abstract operation, like GetBuiltinValue - then you could do "get the first value, if the second is undefined and the first isn't return true, get the second value, compare"